src/main/java/github/weichware10/toolbox/Main.java

```diff
@@ -1,86 +1,83 @@
 package github.weichware10.toolbox;
 
-import github.weichware10.toolbox.gui.ConfirmBox;
-import github.weichware10.toolbox.gui.Startbildschirm;
-import java.io.IOException;
-import java.io.OutputStream;
+import github.weichware10.toolbox.gui.App;
+import github.weichware10.toolbox.gui.dialogs.ConfirmDialog;
+import io.github.cdimascio.dotenv.Dotenv;
 import java.io.PrintStream;
+
+import org.joda.time.DateTime;
+
 import javafx.application.Application;
-import javafx.event.EventHandler;
-import javafx.scene.Scene;
-import javafx.scene.control.TextArea;
+import javafx.geometry.Rectangle2D;
 import javafx.scene.image.Image;
+import javafx.stage.Screen;
 import javafx.stage.Stage;
 import javafx.stage.WindowEvent;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'javafx.stage.WindowEvent' import. Should be before 'org.joda.time.DateTime'.